src/lib/geoip/utils.ts

```diff
@@ -1,5 +1,20 @@
 import anyAscii from 'any-ascii';
 
-export const normalizeCityName = (name: string): string => anyAscii(name).toLowerCase().replace(/(?:\s+|^)(?:the|city)(?:\s+|$)/gi, '');
+/* eslint-disable @typescript-eslint/naming-convention */
+const altNames: Record<string, string> = {
```

Comment:
we should move aliases to a seperate file. Just like we did for ISP names.

https://github.com/jsdelivr/globalping/blob/master/src/lib/location/networks.ts